modules/trusted-firmware-m/tfm_boards/common/config.cmake

```diff
@@ -5,29 +5,31 @@
 #
 
 # Override the platform crypto key handling
-set(PLATFORM_DEFAULT_CRYPTO_KEYS        FALSE       CACHE BOOL      "Use default crypto keys implementation.")
+set(PLATFORM_DEFAULT_CRYPTO_KEYS              FALSE       CACHE BOOL      "Use default crypto keys implementation.")
```

Comment:
Same here. I cannot spot the change as the whole file is about whitespace changes.